src/Cardano/Wallet/AddressDerivation.hs

```diff
@@ -309,3 +310,51 @@ keyToAddress (Key xpub) =
     Address $ CBOR.toStrictByteString $ encodeAddress xpub encodeAttributes
   where
     encodeAttributes = CBOR.encodeMapLen 0
+
+-- $use
+-- 'Key' and 'Index' allow for reprenting public keys, private keys, hardened
+-- indexes and soft indexes for various level in a non-ambiguous manner. Those
```

Comment:
Would it be worth clarifying "soft" → "soft (non-hardened)" ?

Reply:
Nice idea :+1: